static/playGame.html

```diff
@@ -15,22 +15,22 @@
 <div class="container" data-ng-show="!isOpenGame()">
   <div class="eight columns">
     <p data-ng-show="gameIsActive()">
-      You are in an ongoing chess game, the pot is at {{game.pot}}.
+      You are in an ongoing chess game, the pot is at {{game.pot}} Ether.
     </p>
     <p data-ng-show="gameIsWon()">
       You have won this game.
-      <span data-ng-show="getGame().pot > 0">The pot of {{getGame().pot}} is being transferred to you.</span>
-      <span data-ng-hide="getGame().pot > 0">The pot was transferred to you.</span>
+      <span data-ng-show="getGame().pot > 0">The pot of {{getGame().pot}} Ether is being transferred to you.</span>
+      <span data-ng-hide="getGame().pot > 0">The pot of {{getGame().pot}} Ether was transferred to you.</span>
```

Comment:
I added the pot size here, so the users see it after the game has ended. But pot and wonEther are set to zero already, so this does not work. If someone has an idea on how to fix that, that would be nice. Otherwise I change it to the state before.